src/client/datascience/interactive-window/nativeInteractiveWindowCommandListener.ts

```diff
@@ -478,6 +478,7 @@ export class NativeInteractiveWindowCommandListener implements IDataScienceComma
             // Scroll to cell in the one that has the cell. We need this so
             // we don't activate all of them.
             // eslint-disable-next-line @typescript-eslint/prefer-for-of
+            possibles.forEach((item) => item.scrollToCell(id));
```

Comment:
This seems like it would scroll to multiple cells and then go back and scroll to the first one found? Did you intend to leave this here?

Reply:
oops, good catch. was meant to be removed.